PhotoKingdomAPI/PhotoKingdomAPI/Controllers/Manager.cs

```diff
@@ -1053,6 +1035,17 @@ public IEnumerable<AttractionPhotowarWithDetails> checkOwns()
 
                         var addOwn = CreateAttractionOwnForPhotoUpload(firstUpload.Id);
                         if (!addOwn) return null;
+
+                        // Check Photowar queue if photowar ended before 3 days
+                        if (!checkCurrentPhotowar(photowar.AttractionId)
+                            && photowar.EndDate < DateTime.Now.AddDays(-3))
+                        {
+                            var queue = checkPhotowarQuque(photowar.AttractionId);
+                            if (queue != null)
+                            {
+                                createPhotowarFromQueue(queue, firstUpload);
+                            }

```

Comment:
The algorithm is fine for checking queue whenever 3 days has already passed since the photowar ended. But what if it's within the 3 days? Then it will skip checking the queue. And then we won't have another way to check the queue anymore because next time we run CheckOwns, it won't check this attraction anymore because the photowar already has a winner.

Reply:
I know what to do, instead of checking if 3 days has already passed, why don't we remove that condition, and we will always check the queue no matter what, and then if there is a next queue item, we will just create a photowar that is 3 days after the photowar that just ended?

Reply:
It's right. Missing in the case.